packages/replay/README.md

```diff
@@ -1,4 +1,4 @@
-# sentry-replay
+# @sentry/replay
```

Comment:
l: To be in line with our other packages I'd go for something like "Sentry (Session?) Replay"